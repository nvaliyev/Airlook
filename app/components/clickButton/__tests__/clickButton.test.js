import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ClickButton from '..';
describe('clickButton', () => {
    test(' clickButton snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<ClickButton/>);
        expect(snapshot).toMatchSnapshot();
    });
});


