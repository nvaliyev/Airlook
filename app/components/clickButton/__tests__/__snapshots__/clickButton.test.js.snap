// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clickButton  clickButton snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Array [
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "backgroundColor": "#5d6fc4",
        "borderRadius": 5,
        "elevation": 5,
        "margin": 3,
        "marginHorizontal": 25,
        "padding": 20,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 8,
          "width": 0,
        },
        "shadowOpacity": 0.6,
        "shadowRadius": 10,
      },
      Object {
        "backgroundColor": undefined,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#F7F9FC",
        "fontSize": 18,
      }
    }
  >
     
     
  </Text>
</TouchableOpacity>
`;
